README.md

```diff
@@ -39,7 +39,7 @@ import com.softwaremill.sttp.asynchttpclient.cats.AsyncHttpClientCatsBackend
 
 implicit val cs = IO.contextShift(ExecutionContext.fromExecutor(Executors.newCachedThreadPool()))
 implicit val sttpBackend = AsyncHttpClientCatsBackend[cats.effect.IO]()
-val c = Client("scala-sdk-examples", "api.cognite.com")
+val c = new GremlinClient[IO, Nothing]("scala-sdk-examples", "https://api.cognitedata.com")
```

Comment:
GenericClient? :)

Reply:
oh...should have copy-pasted better. 
we should think about using tut-plugin to compile these examples and make sure that they actually work.

Reply:
Totally! @andrews47288 is working on the documentation right now.